code/services/JStatExampleNormalDeviate/JStatExampleNormalDeviate.js

```diff
@@ -1,5 +1,9 @@
-Y_OFFSET = 2
-function Example_NormalDeviate(req, resp){
+/**
+ * This example creates a normal deviate dataset
```

Comment:
link to that API Method

Reply:
https://github.com/rreinold/jstat-statistics-toolkit/pull/1/commits/60c64ee9a362a0d40a1dbe84bd0ed466450ae98c